tests/mocks/HUBComponentMock.h

```diff
@@ -39,7 +41,8 @@ NS_ASSUME_NONNULL_BEGIN
     HUBComponentWithSelectionState,
     HUBComponentViewObserver,
     HUBComponentContentOffsetObserver,
-    HUBComponentActionPerformer
+    HUBComponentActionPerformer,
+    HUBComponentActionObserver
```

Comment:
Suggestion: Make this configurable, so that not **all** mocks conform to this protocol. See how this was done for the other protocols. Otherwise you're essentially only testing the success code path, not the failure one.